drivers/cc110x/cc110x.c

```diff
@@ -175,3 +175,34 @@ int cc110x_set_channel(cc110x_t *dev, uint8_t channel)
     dev->netdev.event_callback(&dev->netdev, NETDEV_EVENT_FHSS_CHANGE_CHANNEL);
     return 0;
 }
+
+int cc110x_wakeup(cc110x_t *dev)
+{
+    if (cc110x_acquire(dev) != SPI_OK) {
+        return -EIO;
+    }
+
+    cc110x_power_on(dev);
```

Comment:
I'm not sure if this is safe to do. The current `cc110x_power_on()` will reconfigure the CS pin as a GPIO to signal the chip to boot up, wait for the upper bound the chip needs to boot, and than perform `spi_init_cs()` again.

Calling `spi_init_cs()` while the SPI interface is acquired might cause issues. Even if all SPI drivers would support this, `cc110x_power_on()` takes ages. So this would block the SPI bus, which might be shared with other devices, for way longer than usual operation. This could, thus, prevent the use of the power saving features in real-time scenarios.

Reply:
Yeah you're right, I'll check what's the current status at this point, to see if a "reinit" is really needed.